tests/recovery/recovery_test/recovery_test.go

```diff
@@ -168,6 +172,172 @@ func TestSnapshotFixInvalidFiles(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// TestConsistencyWhenKill9AfterModify will test the data consistency while it encountered kill -9 signal.
+func TestConsistencyWhenKill9AfterModify(t *testing.T) {
+	// assumption: the test is run on filesystem & not directly on object store
+	dataRepoPath := path.Join(*repoPathPrefix, dirPath, dataPath)
+
+	baseDir := t.TempDir()
+	if baseDir == "" {
+		t.FailNow()
+	}
+
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir, dataRepoPath)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			t.Skip("Skipping crash consistency tests because KOPIA_EXE is not set")
+		} else {
+			t.Skip("Error creating SnapshotTester:", err)
+		}
```

Comment:
You do not need the else clause because Skip() will exit the test.

```suggestion

		t.Skip("Error creating SnapshotTester:", err)
```

Reply:
I deleted the else clause. Thanks~